Makefile

```diff
@@ -26,5 +46,8 @@ container-smoke: $(CONTAINER_ID)
 	docker stop $(shell cat $(CONTAINER_ID))
 	docker logs $(shell cat $(CONTAINER_ID))
 
+prune: clean
+	docker system prune -f
```

Comment:
Nit: This will remove all dangling containers/images/volumes/etc. on the user machine which may be an unexpected side-effect. I would try to limit the "clean" targets so they only remove stuff specific to the project.

For example, if I cloned this repo and ran "make prune" on my machine, it would remove all stopped containers which I might have intended to start again which would be unexpected. At the least it should give a warning or something I think.

Reply:
A good point.  I'll remove this target, as it isn't important to the project.

Reply:
Addressed by 067830ffffc201fb700a220f142541d8221c78fc